### PR TITLE
Change tests to use github repo 

### PR DESCRIPTION
**Pull request by gadotroee**
Also removing the images that we build for gcr

### CODE REVIEW

File: tap/extensions/get-folder-of-tests.sh [hunk 1]
```diff
@@ -0,0 +1,23 @@
+mizu_test_files_repo=https://github.com/up9inc/mizu-tests-data
+mizu_test_files_tmp_folder="mizu-test-files-tmp"
+
+requested_folder=$1
+destination_folder_name=$2
+echo "Going to download folder (${requested_folder}) from repo (${mizu_test_files_repo}) and save it to local fodler (${destination_folder_name}"
```
Comment (RoyIsland) [inline tap/extensions/get-folder-of-tests.sh:6]:
```suggestion
echo "Going to download folder (${requested_folder}) from repo (${mizu_test_files_repo}) and save it to local folder (${destination_folder_name}"
```

File: tap/extensions/get-folder-of-tests.sh [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+git clone ${mizu_test_files_repo} ${mizu_test_files_tmp_folder} --no-checkout --depth 1 --filter=blob:none --sparse
+echo "Cloneing repo to tmp folder (${mizu_test_files_tmp_folder})"
```
Comment (RoyIsland) [inline tap/extensions/get-folder-of-tests.sh:9]:
```suggestion
echo "Cloning repo to tmp folder (${mizu_test_files_tmp_folder})"
```

Comment (gadotroee) [reply tap/extensions/get-folder-of-tests.sh]: https://github.com/up9inc/mizu/pull/1211/commits/710de80520b8e277578a162b1b222dfa35df351f

File: tap/extensions/get-folder-of-tests.sh (hunk 1 above)
Comment (gadotroee) [reply tap/extensions/get-folder-of-tests.sh]: https://github.com/up9inc/mizu/pull/1211/commits/710de80520b8e277578a162b1b222dfa35df351f

**Review by haiut — APPROVED** [review]

**Pull request merged**